src/include/ipc/dai-imx.h

```diff
@@ -13,7 +13,7 @@
 
 /* ESAI Configuration Request - SOF_IPC_DAI_ESAI_CONFIG */
 struct sof_ipc_dai_esai_params {
-	struct sof_ipc_hdr hdr;
```

Comment:
sof_ipc_hdr consists of two 32-bit variables, not just one. Is this intentional?

Reply:
yep, we need 2 * uint32 here.

Reply:
indeed, I looked at the wrong structure.